libs/ngx-charts-on-fhir/src/lib/data-layer-selector/data-layer-selector.component.css

```diff
@@ -0,0 +1,10 @@
+/* Used to make filter responsive for mobile view*/
+.data-layer-selector {
+  display: flex;
+  align-items: flex-start;
+  gap: 8px;
+}
+
+.data-layer-selector mat-button-toggle-group {
+  margin-top: 2px;
+}
```

Comment:
why is this here? data-layer-selector component does not have a mat-button-toggle-group.